bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp

```diff
@@ -58,13 +58,13 @@ void CreateLeggedOdometry(pybind11::module& module)
              py::arg("encoder_speeds"))
         .def("advance", &LeggedOdometry::advance)
         .def("reset_estimator",
-             py::overload_cast<const InternalState&>(&LeggedOdometry::resetEstimator),
+             py::overload_cast<const InternalState&>(&FloatingBaseEstimator::resetEstimator),
              py::arg("new_state"))
         .def("reset_estimator",
              py::overload_cast<const Eigen::Quaterniond&, const Eigen::Vector3d&>(
-                 &LeggedOdometry::resetEstimator),
-             py::arg("new_imu_orientation"),
-             py::arg("new_imu_position"))
+                 &FloatingBaseEstimator::resetEstimator),
+             py::arg("new_base_orientation"),
+             py::arg("new_base_position"))
```

Comment:
Maybe a rebase over the latest master will remove this change?

Reply:
Sorry, there was a typo in the main message, this PR dependend on https://github.com/dic-iit/bipedal-locomotion-framework/pull/380, not https://github.com/dic-iit/bipedal-locomotion-framework/pull/381 as erroneously reported.

Reply:
Anyhow, this is done now.